Sources/SwiftParserDiagnostics/SyntaxExtensions.swift

```diff
@@ -39,6 +39,18 @@ extension UnexpectedNodesSyntax {
       return nil
     }
   }
+
+  /// If this only contains two tokens, one satisfying `firstCondition`, and one satisfying `secondCondition`,
+  /// return these tokens as a tuple, otherwise return `nil`. The function is not order sensitive
```

Comment:
>  The function is not order sensitive

I think it should be order sensitive, that’s the whole point of the function, right? See https://github.com/apple/swift-syntax/pull/1446/files#r1150941427